nekoyume/translations/ja/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,300 @@
+# Japanese translations for PROJECT.
+# Copyright (C) 2018 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2018-08-15 15:01+0900\n"
+"PO-Revision-Date: 2018-08-15 15:02+0900\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```

Comment:
@ziwon Could you add your name and mail information here? (If you don't want to add, I think it is better to remove this line.)

Reply:
@jckdotim Ok, then I'm gonna add it right now.